index.html

```diff
@@ -9,30 +9,46 @@
                 
 </head>
 <body>
+    <section id="main-page">
      <header class="nav-bar">
                 <a class="logo" href="#">Muhammad Nafees</a>
-                <nav class="menu" href="#"><img src="images/Union.png" alt="Menu bar"></nav>
+                <nav class="menu" href="#"><img id="menu-icon" src="images/menu.png" alt="Menu bar">
+                <div id="nav-menu">
+                    <a href="#">Portfolio</a>
+                    <a href="">About</a>
+                    <a href="">Contact</a>
+                    <a href=""><img src="images/mail-icon.png" alt=""></a>
+                </div></nav>
      </header>
         <section class="content">
+            <div class="main-text">
     <h1 class="my-name">Hey There.<br/> I'm Nafees</h1>
     <h3 class="profession">I’m a Software Developer</h3>
     <p class="i-can-do">I can help you build a product , <br /> feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
         don’t hesitate to contact me.</p>
+    </div>
         <ul class="links">
             <li><a title="GitHub" href="https://github.com/ngeelani48"><img src="images/github.png" alt="GitHub Profile"></a></li>
             <li><a title="Twitter" href="https://twitter.com/ngeelani48"><img src="images/twitter.png" alt="Twitter Profile"></a></li>
             <li><a title="Medium" href="https://medium.com/@engnrnafees"><img src="images/medium.png" alt="Medium Profile"></a></li>
             <li><a title="LinkedIn" href="https://www.linkedin.com/in/m-n-geelani-3446097275/"><img src="images/Linkedin icon.png" alt="LinkedIn Profile"></a></li>
             <li><a title="Angelist" href="#"><img src="images/Vector.png" alt="angelist Profile"></a></li>
-        </ul>
+        </ul>  
     </section>
+</section>
     <section class="project-section">
+        <div class="my-recent-works">
         <div class="my-work-title">
             <h1>My Recent Works</h1>
         </div>
         <div id="line"></div>
+    </div>
         <div id="works-section">
             <div class="project1">
+                <div class="project1-picture">
+                    <img src="images/Project 1.png" alt="">
+                </div>
+                <div class="project1-content">
                 <h1 class="project1-title">Multi-Post-Stories</h1>
                 <p class="project1-description"> A daily selection of privately personalized reads; no accounts or
                     sign-ups required. has been the industry's standard dummy text
```

Comment:
There are some differences between your project and the design as regards this section😉
kindly take note of the highlighted areas and the spacings🤗 I suggest your image and the text should occupy 50% of each of the containers as specified in the design🤗

your project
<img width="938" alt="your own" src="https://user-images.githubusercontent.com/64643995/213491777-53e97f15-6ca8-443e-96a6-436451facb44.png">

figma
<img width="515" alt="display project" src="https://user-images.githubusercontent.com/64643995/213491875-54d33854-8d28-41f6-b381-10c4955f8cfb.png">